buildSrc/subprojects/parser/src/main/java/org/apache/calcite/buildtools/parser/DialectTraverser.java

```diff
@@ -76,13 +76,16 @@ public ExtractedData extractData() {
   }
 
   /**
-   * Generates the parserImpls.ftl file for the dialect.
+   * Generates the parserImpls.ftl file for the dialect. It is assumed that
+   * there exists a path src/resources/license.txt at the root parsing directory
+   * which was specified in the constructor.
    *
    * @param extractedData The extracted data to write to the output file
    */
   public void generateParserImpls(ExtractedData extractedData) {
-    Path outputFilePath = Paths.get(outputPath).toAbsolutePath();
-    Path licensePath = Paths.get("src", "resources", "license.txt");
+    Path outputFilePath = dialectDirectory.toPath().resolve(outputPath);
```

Comment:
@michaelvdow Is this the expected behaviour? To give an example if the dialect id bigquery the outputFilePath would be `parsing/dialects/bigquery/$outputPath`

Reply:
Yeah that looks like the right path